### PR TITLE
Add noscript page

### PR DESCRIPTION
**Pull request by JustusFT**
OP#2321

### CODE REVIEW

File: client/src/public/index.html [hunk 1]
```diff
@@ -4,17 +4,86 @@
...
+          <p>
+            In order to use Winden, your web browser must have JavaScript
+            enabled.
+          </p>
+          <p>Please refresh the page after enabling JavaScript.</p>
+          <p>
+            For instructions on how to enable JavaScript,
+            <a
+              href="https://www.enable-javascript.com/"
+              target="_blank"
```
Comment (donpui) [inline client/src/public/index.html:79]: We can remove _blank, to leave decision for User.

**Comment by donpui** [comment]
@JustusFT do you want to add something more here? From my side seems ok to make PR and merge

File: client/src/public/index.html [hunk 2]
```diff
@@ -4,17 +4,86 @@
...
+      }
+      .ns-content {
+        background-color: #fff;
+        border: 1px solid #aaa;
+        padding: 20px;
+        border-radius: 20px;
+      }
+    </style>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
```
[the remark this answers is not in the document]
Comment (JustusFT) [reply client/src/public/index.html:53]: @donpui I will replace the cdn links before I un-draft the PR

[the remark this answers is not in the document]
Comment (donpui) [reply client/src/public/index.html:53]: We don't use CDN

[the remark this answers is not in the document]
Comment (JustusFT) [reply client/src/public/index.html]: Removed external resources

**Review by donpui — APPROVED** [review]
Looks good and works.

File: client/src/public/index.html [hunk 3]
```diff
@@ -22,22 +22,54 @@
...
+        font-style: normal;
+      }
+
+      body {
+        background: linear-gradient(222.19deg, #f0f0f0 23.77%, #ffffff 98.02%);
+        font-size: 16px;
+        color: #282f39;
+        min-width: 320px;
+      }
+
```
Comment (meejah) [inline client/src/public/index.html:65]: Do we really need all 5 variants of this font? Do we need a new font here at all?

Comment (JustusFT) [reply client/src/public/index.html:65]: I included the variants to follow the style guide/design docs for winden. AFAIK all weights except for 500 are used right now.

**Pull request merged**